Error when using `Automatic` or `Managed` tool provider

```
Error: Cannot find module 'which'
Require stack:
- /home/mike/.vscode-oss/extensions/edacation.edacation-0.5.10-universal/dist/extension/extension.js
- /nix/store/8dqzch2p9myp5gayrvgvwdljnzqch2cg-vscodium-1.88.1.24104/lib/vscode/resources/app/out/vs/loader.js
- /nix/store/8dqzch2p9myp5gayrvgvwdljnzqch2cg-vscodium-1.88.1.24104/lib/vscode/resources/app/out/bootstrap-amd.js
- /nix/store/8dqzch2p9myp5gayrvgvwdljnzqch2cg-vscodium-1.88.1.24104/lib/vscode/resources/app/out/bootstrap-fork.js
        at Module._resolveFilename (node:internal/modules/cjs/loader:1084:15)
        at i._resolveFilename (node:electron/js2c/utility_init:2:13405)
        at Module._load (node:internal/modules/cjs/loader:929:27)
        at c._load (node:electron/js2c/node_init:2:13672)
        at d._load (/nix/store/8dqzch2p9myp5gayrvgvwdljnzqch2cg-vscodium-1.88.1.24104/lib/vscode/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:175:6922)
        at i._load (/nix/store/8dqzch2p9myp5gayrvgvwdljnzqch2cg-vscodium-1.88.1.24104/lib/vscode/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:172:36297)
        at t._load (/nix/store/8dqzch2p9myp5gayrvgvwdljnzqch2cg-vscodium-1.88.1.24104/lib/vscode/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:137:44886)
        at Module.require (node:internal/modules/cjs/loader:1150:19)
        at h (/nix/store/8dqzch2p9myp5gayrvgvwdljnzqch2cg-vscodium-1.88.1.24104/lib/vscode/resources/app/out/vs/loader.js:4:647)
        at n (/home/mike/.vscode-oss/extensions/edacation.edacation-0.5.10-universal/dist/extension/extension.js:1:78138)
        at t.which (/home/mike/.vscode-oss/extensions/edacation.edacation-0.5.10-universal/dist/extension/extension.js:1:78351)
        at g.getExecutionOptions (/home/mike/.vscode-oss/extensions/edacation.edacation-0.5.10-universal/dist/extension/extension.js:1:137355)
        at y.getToolProvider (/home/mike/.vscode-oss/extensions/edacation.edacation-0.5.10-universal/dist/extension/extension.js:1:137831)
        at y.getName (/home/mike/.vscode-oss/extensions/edacation.edacation-0.5.10-universal/dist/extension/extension.js:1:137571)
        at f.execute (/home/mike/.vscode-oss/extensions/edacation.edacation-0.5.10-universal/dist/extension/extension.js:1:127656)
        at t.TaskTerminal.executeNextTask (/home/mike/.vscode-oss/extensions/edacation.edacation-0.5.10-universal/dist/extension/extension.js:1:132133)
        at async t.TaskTerminal.open (/home/mike/.vscode-oss/extensions/edacation.edacation-0.5.10-universal/dist/extension/extension.js:1:130493)
```

--- a/src/common/node-modules.ts
+++ b/src/common/node-modules.ts
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
 /* eslint-disable @typescript-eslint/consistent-type-imports */
 /* eslint-disable @typescript-eslint/no-unsafe-return */
 /* eslint-disable @typescript-eslint/no-unsafe-call */
@@ -37,6 +38,6 @@ export const fs = () => requireModule('fs') as ModuleFS;
 export const os = () => requireModule('os') as ModuleOS;
 export const process = () => requireModule('process') as ModuleProcess;
 export const stream = () => requireModule('stream') as ModuleStream;
-export const which = () => requireModule('which') as ModuleWhich;
+export const which = () => require('which') as ModuleWhich;
 export const workerThreads = () => requireModule('worker_threads') as ModuleWorkerThreads;
 export const zlib = () => requireModule('zlib') as ModuleZLib;

--- a/webpack.config.mjs
+++ b/webpack.config.mjs
@@ -55,7 +55,8 @@ const extensionConfig = Object.assign({}, baseConfig, {
     },
     externals: {
         vscode: 'commonjs vscode',
-        fs: 'fs'
+        fs: 'fs',
+        'fs/promises': 'fs/promises'
     },
     module: {
         rules: [

